### DISCUSSION TITLE
Bromite, Vanilla Chromium, Ungoogled Chromium Crash on Launch - LineageOS 12.1

### DISCUSSION THREAD

**Discussion by AlbusPercivalDumbledore** [bromite/bromite#1680] [Q&A] [answered]
Hi, this issue isn't specific to Bromite. Yet, I couldn't find a solution for it anywhere, so decided to ask here.

I'm trying to install Bromite v95.0.4638.79 (last supported version for Lollipop) on LineageOS 12.1, on Kindle Fire Tablet. The arm 32 apk (arm_ChromePublic.apk) installs alright, yet the app crashes on launch. I'm facing the same issue with Ungoogled Chromium & Vanilla Chromium.

I haven't installed Google Play Services/ Play Services framework on LineageOS. Am I required to install them for Bromite to function?

**Comment by csagan5** [comment, marked as answer]
Bromite does not use Play Services; you should look into the crash log to figure out which component of LineageOS is crashing. You should better ask in the LineageOS community.

**Reply by AlbusPercivalDumbledore** [reply]
I'll ask there then. Thanks!
